### DISCUSSION TITLE
Pitch Location Analysis for Shohei Ohtani in the 2023 Season

### DISCUSSION THREAD

**Discussion by StuffbyYuki** [has2k1/plotnine#822] [2024 Plotnine Contest]
### Authors

Yuki Kakegawa

### Links

[linkedin](https://www.linkedin.com/in/yukikakegawa/), [consulting company](https://www.oremdata.com/), [personal website](https://yukikakegawa.me/), [medium](https://medium.com/@yukithejapanese)

### Full description

Scatter plots showing pitch locations for the pitches thrown in the 2023 regular season by Shohei Ohtani. Plots show strikes/balls per pitch type and batter's stand. 

### Code repository

https://github.com/StuffbyYuki/Plotnine-Contest/blob/main/2024/shohei_ohtani_analyses/pitching_analysis_2023.ipynb

**Comment by has2k1** [comment]
![pa-2023](https://github.com/user-attachments/assets/5caac43c-3cd2-4b54-9765-26e46b272bd0)
<hr/>

![pa-2023-2](https://github.com/user-attachments/assets/0827426e-81ea-46af-9991-c5e27433bb45)

**Comment by has2k1** [comment]
@StuffbyYuki, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you again for taking part in the contest.
